src/Microsoft.TestPlatform.PlatformAbstractions/Microsoft.TestPlatform.PlatformAbstractions.csproj

```diff
@@ -31,10 +31,10 @@
     <Compile Include="Interfaces\**\*.cs" />
     <Compile Include="Properties\**\*.cs" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <Compile Include="netstandard1.0\**\*.cs" />
```

Comment:
>netstandard1.0 [](start = 22, length = 14)

should we rename the folder as just netstandard ?

Reply:
Yes. Done